### PR TITLE
Create laser-and-photonics-reviews.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/69427/style-request-laser-photonics-reviews

### CODE REVIEW

**Comment by rmzelle** [comment]
Thanks!

File: laser-and-photonics-reviews.csl [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout vertical-align="sup" delimiter="," prefix="[" suffix="]">
```
Comment (qwerty) [inline laser-and-photonics-reviews.csl]: @rmzelle There should be no superscript in the "citation-number":
` <layout delimiter="," prefix="[" suffix="]">`

Comment (POBrien333) [reply laser-and-photonics-reviews.csl]: I've just checked this paper for example and the number is superscripted: http://onlinelibrary.wiley.com/doi/10.1002/lpor.201700082/epdf

Comment (qwerty) [reply laser-and-photonics-reviews.csl]: Yes you are right, my apologies: it seems they have changed the layout since July of last year.

**Pull request merged**